cmd/controller-manager/app/options/options.go

```diff
@@ -53,6 +53,9 @@ type Options struct {
 	NSAutoPropExcludeRegexp string
 	CreateCRDsForFTCs       bool
 	ClusterJoinTimeout      time.Duration
+
+	MaxPodListers        int64
+	EnablePodSimplifying bool
```

Comment:
```suggestion
	EnablePodSimplification bool
```

Reply:
Pls rmb to change all instances of this flag

Reply:
"simplification" is too long, what about "pruning"?

Reply:
Prefer "pruning"